schemas/v1.0/azure.yaml.json

```diff
@@ -239,6 +239,28 @@
                         "github",
                         "azdo"
                     ]
+                },
+                "variables": {
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as variables.",
+                    "description": "If variable is found on azd environment, it is set as a variable for the pipeline.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "secrets": {
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as secrets.",
+                    "description": "If variable is found on azd environment, it is set as a secret for the pipeline.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "additionalVariablesAsSecrets": {
```

Comment:
This seems like overkill to me to add this feature.  I also don't think the name matches the description/functionality.

The name looks like "add these" the description states "add all vars as secrets"

Reply:
name is not final.  See: https://github.com/Azure/azure-dev/pull/3155#issuecomment-1876179447